packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -541,10 +541,26 @@ export class StripeWebhookHandler extends StripeHandler {
         this.stripeHelper.getCustomerPaypalAgreement(customer);
       // The customer needs to be updated for their name.
       if (billingAgreementId) {
-        await this.paypalHelper.updateStripeNameFromBA(
-          customer,
-          billingAgreementId
-        );
+        try {
+          await this.paypalHelper.updateStripeNameFromBA(
+            customer,
+            billingAgreementId
+          );
+        } catch (err) {
+          if (
+            err.output &&
+            err.output.payload.data.message ===
```

Comment:
We can check the error type more reliably by looking at the errno on it and seeing if its `ERRNO.INTERNAL_VALIDATION_ERROR`. This is less brittle because the message could change later in its text, but the errno will still match this.